### DISCUSSION TITLE
Digital signal generator using captured trace

### DISCUSSION THREAD

**Discussion by Ashram56** [gusmanb/logicanalyzer#44] [Ideas]
One idea which I think might be useful (at least it is in the scope of what I'm attempting to do):

It would be nice to be able to use captured trace as a source for a digital signal generator and another Pico. Essentially allowing to replicate on bench the behaviour of a specific bus.

In my specific cases, I can think of quite a few instances where this would be extremely useful.

**Comment by gusmanb** [comment]
You aren't the first one that asks for this, there is an open issue for this functionality: https://github.com/gusmanb/logicanalyzer/issues/40

Implementing the code is relatively easy but I need to design a new shifter board as the one that I have created is only for input and for this to work it must allow to be set as input or output.
I'm waiting for some level shifters for other project that their direction is selectable, if I get a good result with them I will create a new shifter board and implement this functionality.

Cheers!
